_data/icons/polynomial.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url": "ipfs://bafybeiadb4izxxptk3wy34cam3k2rxgqb2vrzxtltqnp3vwdvd2igwoghu",
+      "width": 500,
+      "height": 500,
+      "format": "png"
```

Comment:
error due to the expected format and the format from url is not match 

change this format to jpg